src/main/java/com/gluonhq/substrate/util/Strings.java

```diff
@@ -73,4 +76,12 @@ public static String substitute(String template, Map<String,String> context) {
         return substitute(template, context::get);
     }
 
+    /**
+     * Converts a string of comma separated strings into an array of strings
+     * @param s a string that contains comma separated strings
+     * @return a list of strings
+     */
+    public static List<String> splitString(String s) {
```

Comment:
Good decision! I totally forgot about this one while creating the Strings class.